src/main/java/com/thebois/views/IProjector.java

```diff
@@ -0,0 +1,20 @@
+package com.thebois.views;
+
+import com.badlogic.gdx.math.Vector2;
+
+/**
+ * Projector used to convert space and world coordinates.
+ */
+public interface IProjector {
+
+    /**
+     * Unprojects screen coordinates to world coordinates.
+     *
+     * @param screenCordX X coordinate of a position in screen space.
+     * @param screenCordY Y coordinate of a position in screen space.
+     *
+     * @return Returns a Vector2 in the world coordinates.
+     */
+    Vector2 unproject(int screenCordX, int screenCordY);
```

Comment:
This is no big deal at all since it's clear anyway, but we talked about writing the whole name, so `screenCoordinateX`. Or did we mean just not to write for example `sCX`?

Reply:
Probably best to not have any exceptions, easier to remember that way. Writing out coordinate here is better.

Reply:
On my todo list, will do it tomorrow